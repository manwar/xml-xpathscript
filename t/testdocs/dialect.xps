Hello        <%- $x = 1 %> world
Hello <% $x = 1 -%> world
Hello <%- $x = 1 -%>
world
Hello        <%-= ":" -%>        world

<%#
    invisible

    fdgdssfd

    die die die








%>

<%~ //foo %>
