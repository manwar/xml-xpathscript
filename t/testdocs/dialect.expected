Hello world
Hello world
Helloworld
Hello:world



<foo></foo>
